apps/etterlatte-trygdetid/src/main/resources/logback.xml

```diff
@@ -5,7 +5,16 @@
         </encoder>
     </appender>
 
+    <appender name="STDOUT_JSON" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder class="net.logstash.logback.encoder.LogstashEncoder" />
+    </appender>
+
     <root level="INFO">
-        <appender-ref ref="STDOUT"/>
+        <appender-ref ref="STDOUT_JSON"/>
```

Comment:
Ville også lagt til en logback.xml for lokal kjøring, da uten stdout_json. Se de andre appene for ref.